src/EnergyPlus/EconomicTariff.cc

```diff
@@ -2898,11 +2898,6 @@ namespace EconomicTariff {
         Real64 annualAggregate;
         int annualCnt;
 
-        if (!ioFiles.outputControl.tabular) {
-            WriteTabularFiles = false;
```

Comment:
@mbadams5 If I'm following this correctly, one side effect of these changes is that `WriteTabularFiles` will be `true` if there are any triggers in the idf file regardless of the output file settings. This flag is used to skip a bunch of timestep calls to [gather functions here](https://github.com/NREL/EnergyPlus/blob/b1c7a5b4634ed662ef93c23b195ba084a09df3c6/src/EnergyPlus/OutputReportTabular.cc#L792-L806). 

Is there a place after all of the Output* input processing is complete that we could test if "Output Tabular = No" and "Output SQLite = No" then set `WriteTabularFiles = false` ?

Reply:
Yeah, that is an interesting idea. I think that is possible and I'll try it out.